finger-exercises-react/react/ecmascript/01 - spread/index.js

```diff
@@ -1,8 +1,12 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (isArray(args)) {
+    if (args.length > 1) {
+      return Math.min(args);
+    }
+  }
+  return args[0];
```

Comment:
```
if(isArray(args) && args.length) return Math.min(args)
```

Reply:
You could also use an arrow function and do something like this
`export const min = args => isArray(args) && args.length ? Math.min(args) : args[0]`

Reply:
Perfect! I wasn't sure if it was a good practice, here at wolox, to write everything in a single line of code whenever it's possible. This looks way more cleaner without the braces.